### PR TITLE
Pre-commit integration

### PR DESCRIPTION
**Pull request by PetrushynskyiOleksii**
Solve #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Django-girls-blog-PetrushynskyiOleksii
```
Comment (webknjaz) [inline README.md:1]: Why did you remove space next to hash?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pre-commit
```
Comment (webknjaz) [inline requirements.txt]: Don't you afraid of some next version breaking everything?

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PetrushynskyiOleksii) [reply README.md:1]: @webknjaz oh, my bad. Fixed

**Review by webknjaz — APPROVED** [review]
